requirements.txt

```diff
@@ -26,3 +26,4 @@ tqdm==4.36.1
 pyzmq==18.1.1
 mock
 google-auth
+molecule<=2.22
```

Comment:
@14rcole Please move molecule to test dependencies in 
https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/setup.py#L56